development/db-schema.md

```diff
@@ -1,158 +1,150 @@
-
-![User types, in UI for editing/creating users](img/user_types.png)
+```js
+{
+  type: 'person',
+  name: 'A patient',
+  parent: {
+    _id: 'clinic-id',
+    parent: {
+      _id: 'health_centre-id',
+      parent: {
+        _id: 'district_hospital-id'
+      }
+    }
+  }
+}
+```
 
-### People users
+Generally when contactables are **used** in the app they are first "hydrated", with the rest of the information filled in from their parent's place documents:
 
-Note: not all people interacting with Medic Mobile can log into the app!
-E.g. in an SMS project, CHWs are SMS-only and never log into the app. There is no user associated to them. In an android project, they do log into the android app, so they have an associated user.
+```js
+{
+  type: 'person',
+  name: 'A patient',
+  parent: {
+    _id: 'clinic-id',
+    name: 'A clinic',
+    reported_date: 1234,
+    ... // etc
+    parent: {
+      _id: 'health_centre-id',
+      name: 'A Health Centre',
+      reported_date: 1134,
+      ... // etc
+      parent: {
+        _id: 'district_hospital-id',
+        name: 'THE District Hospital',
+        reported_date: 1034,
+        ... // etc
+      }
+    }
+  }
+}
+```
 
-Most People users are **associated with a place** (in the UI: “Restricted to their place” users). They can only access docs within that place.
-E.g. a CHW is restricted to their CHW area, and can only see Family members belonging to their area, and the reports from these family members.
+## Reports
 
-![DB docs representing persons](img/persons.png)
+Reports are created by users filling out and submitting forms, as well as sending in SMS.
 
-Users are represented by a **set of 3 docs** : a `type:person`, a `type:user-settings` and a `type:user`. The `type: user` doc is not in the main `medic` db, it's in the `_users` db (check out couchdb documentation about `_users` db)
+All reports:
+ - Use the `data_record` type
+ - Have their fields stored in the `fields` property
+ - Have the report author's phone number (if it existsz) stored in the `from` field
+ - Store the form's identifier in the `form` field
+ - May have a `contact` property, which is a dehydrated version of the report author's contact and its hierarchy (see above)
 
-The user doc and the user-settings doc share the same `_id` field : `org.couchdb.user:<username>`.
-There is similar information in both.
-The `_user` doc holds the couchdb credentials. It lives on the couch server only.
-The `user-settings` doc is replicated to the clients, who can edit it (so can't be relied on for auth).
+Reports also have some kind of patient identifier, which describes which `person` this report is _about_. This value can be in a few different places:
+ - The patient's shortcode may be found at `doc.patient_id` or `doc.fields.patient_id`
+ - A patient record's `_id` may be found at `doc.patient_uuid` or `doc.fields.patient_uuid`, as well as potientially in the same locations as the shortcode.
```

Comment:
You can also have a report about a place with a `place_id`. I don't think `place_uuid` is a thing...